adafruit_is31fl3731/__init__.py

```diff
@@ -244,9 +244,9 @@ def fade(
         default), the breath feature is used for fading.  if fade_in is None, then
         fade_in = fade_out.  If fade_out is None, then fade_out = fade_in
 
-        :param fade_in: int positive number; 0->100
-        :param fade-out: int positive number; 0->100
-        :param pause: int breath register 2 pause value
+        :param fade_in: fade time in ms, range = 26 to 3328
+        :param fade-out: fade time in ms, range = 26 to 3328
```

Comment:
Maybe there was an intention originally to make the API use "cleaner" numbers 0-100 rather than the literal ms values of 26-3328, though in the way it's implemented now your fix to the docs does appear to reflect the correct values that the implementation is expecting.

Honestly though I think it would be better to change the actual functionality to just accept the values 0-7 and the docstrings can list out how many ms each of them correspond to. 

By making the API accept 26 to 3328 it makes it seem to the developer using it that they are free to choose any value within that range and that the duration would be set to the time they chose. But in reality it seems the actual fade duration is always going to be one of these values if I am understanding the datasheet properly:

```
26
52
104
208
416
832
1664
3328
```

I think it would be more clear to the developer using the API if we only allow 8 possible different values if that is technically how many different possible values are supported by the chip.